vbr/vbrCore/functions/wang2006.m

```diff
@@ -0,0 +1,59 @@
+function [ VBR ] = wang2006( VBR )
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  %
+  % [ VBR ] = wang2006( VBR )
+  %
+  % Conductivity of synthetic polycrystaline Ol (hydrous and Anhydrous)
+  %
+  % Parameters:
+  % ----------
+  % VBR    the VBR structure
+  %
+  % Output:
+  % ------
+  % VBR    the VBR structure, with VBR.out.electric.wang2006_ol structure
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+  % read in eletric parameters
+  ele = VBR.in.electric.wang2006_ol;
```

Comment:
Since you removed the nested structure (nice!) you'll need to make these structure names match the method name in `Params_Electric`: 
```suggestion
  ele = VBR.in.electric.wang2006;
```